docs/application/vstools/install.md

```diff
@@ -19,8 +19,8 @@ To work with Visual Studio Tools for Tizen, your computer must have:
 - Visual Studio 2017 to use Tizen 4.0 and 5.0
 - Visual Studio 2019 to use Tizen 4.0 and higher
 - Visual Studio 2022 to use Tizen 6.5 and higher
-- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed through Tizen package manager.
-- Make sure to set the Tool Path (Tizen SDK) in Tools > Options > Tizen > Tools with installed Tizen Studio path. Also, ensure to set the Google Chrome Path for Tizen Web App Debugging support.
+- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](https://docs.tizen.org/application/tizen-studio/setup/update-sdk/) through Tizen package manager.
```

Comment:
Also, please use relative path to the pages in tizen-docs repository.

Reply:
@mijin85cho, I believe this should be the correct path.
```suggestion
- Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](../tizen-studio/setup/update-sdk.md) through Tizen package manager.
```

Reply:
Thank you @omar-srbd, I have applied this modification [3de2929](https://github.com/Samsung/tizen-docs/pull/1607/commits/3de2929a507705919aaac7f8993d1128f0c97ae3).

Reply:
@kusin10 Thanks for the update!

@omar-srbd Correct. As you can see in see in https://github.com/Samsung/tizen-docs/blob/3de2929a507705919aaac7f8993d1128f0c97ae3/docs/application/vstools/install.md, 
when you click the link, it goes to https://github.com/Samsung/tizen-docs/blob/3de2929a507705919aaac7f8993d1128f0c97ae3/docs/application/tizen-studio/setup/update-sdk.md now.